rundeckapp/grails-app/services/rundeck/services/ScheduledExecutionService.groovy

```diff
@@ -876,6 +876,7 @@ class ScheduledExecutionService implements ApplicationContextAware, Initializing
     def rescheduleJob(ScheduledExecution scheduledExecution, wasScheduled, oldJobName, oldJobGroup, boolean forceLocal, boolean remoteAssigned = false) {
         if (jobSchedulesService.shouldScheduleExecution(scheduledExecution.uuid) && shouldScheduleInThisProject(scheduledExecution.project)) {
             try {
+                deleteJob(scheduledExecution.generateJobScheduledName(), scheduledExecution.generateJobGroupName())
```

Comment:
I'm not sure why this "deleteJob" was added.  Eventually the `registerOnQuartz` method will be called with the job, and the old job will be removed via this line https://github.com/rundeck/rundeck/blob/d65c02cd001612f77f7aab541166384fc2046df2/rundeckapp/grails-app/services/rundeck/services/ScheduledExecutionService.groovy#L4510-L4510

Reply:
We were aware of this but was still added as a preemptive measure.